libs/styles/styles.css

```diff
@@ -12,6 +12,7 @@
 
 @import url('variables.css');
 @import url('typography.css');
+@import url('lazy-styles.css');
```

Comment:
Im guessing we dont want to include lazy-styles in here? This is called separately after LPC